tests/neg/i22193.scala

```diff
@@ -0,0 +1,32 @@
+
+def fn2(arg: String, arg2: String)(f: String => Unit): Unit = f(arg)
+
+def fn3(arg: String, arg2: String)(f: => Unit): Unit = f
+
+def test1() =
+
+  // ok baseline
+  fn2(arg = "blue sleeps faster than tuesday", arg2 = "the quick brown fox jumped over the lazy dog"): env =>
+    val x = env
```

Comment:
this looks valid. does `neg` require failing parts to be marked with `error`?

Reply:
yes, that is a baseline example of what works without controversy.